docker/docker-compose.yml

```diff
@@ -741,7 +741,7 @@ services:
       POSTGRES_DB: ${MG_UI_DB_NAME}
       MG_POSTGRES_MAX_CONNECTIONS: ${MG_POSTGRES_MAX_CONNECTIONS}
     ports:
-      - 6007:5432
+      - 6042:5432
```

Comment:
Where does port `6042` come from?

Reply:
It's the correct UI-db port.

Reply:
Correct UI port is default Postgres port at `5432`. The host port is really any port we chose, and the reason we have this mapping is to expose UI DB so we can debug and deploy UI locally. We used `6xxx` ports for those purposes, but `6042` seems out of place.